src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -51,10 +50,9 @@ export class GithubAppsLogsComponent implements OnInit {
     @Inject(MAT_DIALOG_DATA) public matDialogData: { userId?: number; organization?: string },
     private lambdaEventsService: LambdaEventsService,
     private matSnackBar: MatSnackBar,
-    private descriptorLanguageService: DescriptorLanguageService
+    private mapPipe: MapFriendlyValuesPipe
   ) {
     this.datePipe = new DatePipe('en');
-    this.mapPipe = new MapFriendlyValuesPipe(this.descriptorLanguageService);
```

Comment:
Noticed this when adding PlatformPartnerPipe to map-friendly-values because it caused a compilation error. In Angular, pipes should be injected, not instantiated directly (except for tests).

Reply:
But not sure how to deal with the one above it, so left it. :)